src/bio/bio_xstream.c

```diff
@@ -1658,6 +1658,9 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id, bool self_polling)
 		/* No Meta/WAL blobstore if Metadata on SSD is not configured */
 		if (st != SMD_DEV_TYPE_DATA && !bio_nvme_configured(SMD_DEV_TYPE_META))
```

Comment:
Shall we also update this check, as Metadata on SSD is not configured, it is using PMem, then, Data and Bulk_Data (newly added) is supported, so just check st != TYPE_DATA is missing the TYPE_BULK?

Shall be

(st == TYPE_META || st == TYPE_WAL) && !bio_nvme_configured(TYPE_META)

Reply:
seems it should be like this.